docs/development-guide/ReadWriteSplittingPluginPerformanceResults.md

```diff
@@ -0,0 +1,8 @@
+# Read-Write Splitting Plugin Performance Results
+
+## Benchmarks
+| Benchmark                                                     | Score       | Units |
+|---------------------------------------------------------------|-------------|-------|
+| No Plugin Enabled                                             | 1,105,514.4 | ms/op |
```

Comment:
Are there any graphs like the ones we have for efm and failover performance tests?

Reply:
done